scanners/headless/local_bridge.js

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env node
+
+/******
+* Part of a Python<->Node bridge for local scans.
+*
+* Executed by local_bridge.py, to send scan parameters and
+* receive scan responses over the CLI and STDOUT.
+*******/
+
+const puppeteer = require('puppeteer');
+var base = require("./base")
+
+// the first argument passed in is the name of the scanner
+const scanner = require("../" + process.argv[2]);
+
+// the second argument passed is (ordinarily) JSON-serialized data
+var params;
+
+// Hook to allow slightly easier debugging.
+// TEST_LOCAL=1 ./scanners/headless/local_bridge.js third_parties example.com
+if (process.env.TEST_LOCAL) {
```

Comment:
In normal use, `local_bridge.py` sends a serialized JSON blob to pass a dict of data to `local_bridge.js`. To allow for easier local testing of the local JS code path, this adds a simplistic version of the CLI API that just takes a domain name and assumes a bunch of defaults, if the `TEST_LOCAL` env var is present.